core/src/main/resources/jenkins/install/SetupWizard/index.jelly

```diff
@@ -2,7 +2,6 @@
 <l:html norefresh="true" title="${it.displayName}" xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:main-panel>
     <div class="plugin-setup-wizard-container"></div>
-    <script src="${resURL}/${j.installWizardPath}.js" type="text/javascript"/>
-    <link rel="stylesheet" href="${resURL}/${j.installWizardPath}.css" type="text/css" />
+    <st:adjunct includes="${j.installWizardPath}" />
```

Comment:
@recena well we need @kzantow to verify this since he was the one that introduced the 2 things. I'm not second guessing him and am working off the assumption that he did it for good reason. Doing it this way does not fundamentally change anything he has already done i.e. hopefully fixing the problem we are seeing without introducing a different problem.

If @kzantow verifies that it was never needed and can be removed, then I'm fine with that.